sessions/react-state/demo-food-order-start/src/styles.css

```diff
@@ -6,8 +6,7 @@
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
-    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-family: system-ui;
 }
 
 #root {
```

Comment:
Is this correct?
```suggestion
:root {
```

Reply:
yea i think this selects the `#root` element from CSB